manifests/m/Microsoft/WindowsTerminal/1.18.2681.0/Microsoft.WindowsTerminal.installer.yaml

```diff
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.2.10 $debug=QUSU.CRLF.5-1-19041-3031.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.WindowsTerminal
+PackageVersion: 1.18.2681.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.19041.0
+InstallerType: msix
+Scope: user
+InstallModes:
+- silent
+Commands:
+- wt
+PackageFamilyName: Microsoft.WindowsTerminal_8wekyb3d8bbwe
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.UI.Xaml.2.8
+      MinimumVersion: 2.8.4
```

Comment:
Weren't we meant to change this? I remember @yao-msft's PR had changes requested and got closed as stale
- https://github.com/microsoft/winget-pkgs/pull/111968

Reply:
this is the actual min version we depend on in Terminal; is there something we need to do differently?

Reply:
oh i see, this versus 8.xxxx.yyyy.zzzz

Reply:
I opened #120909 to replace the one Yao had that was stale; 
The `8.2306.22001.0` change is needed for winget to detect that it is installed.

The `2.8.4` will work both before and after #120909 is merged, but should be updated afterwards